packages/react-navigation/src/rum/instrumentation/DdRumReactNavigationTracking.tsx

```diff
@@ -26,29 +27,34 @@ export type ViewNamePredicate = (route: Route<string, any | undefined>, trackedN
 export class DdRumReactNavigationTracking {
 
     private static registeredContainer: NavigationContainerRef | null;
-    
+
     private static navigationStateChangeListener: NavigationListener;
 
     private static appStateListener: AppStateListener;
 
     private static viewNamePredicate: ViewNamePredicate;
 
+    static ROUTE_UNDEFINED_NAVIGATION_WARNING_MESSAGE = "A navigation change was detected but the RUM ViewEvent was dropped as the route was undefined."
+    static NULL_NAVIGATION_REF_ERROR_MESSAGE = "Cannot track views with a null navigationRef."
+    static NAVIGATION_REF_IN_USE_ERROR_MESSAGE = "Cannot track new navigation container while another one is still tracked. Please call `DdRumReactNavigationTracking.stopTrackingViews` on the previous container reference."
+    static ROUTE_UNDEFINED_STATE_LISTENER_ERROR_MESSAGE = "Cannot register an application state listener because the current route is undefined."
```

Comment:
I think we need to change the message here. Listener was registered, it is just the `route` is `undefined`, so we cannot handle it. We just need a bit different wording to make this message different from `ROUTE_UNDEFINED_NAVIGATION_WARNING_MESSAGE` (different context and different call site).